Adding airgap installation in README (#935)
dapr/cli: README.md

```bash
# Example of pulling images from a private registry.
dapr init --image-registry example.io/<username>
```

#### Install in airgap environment

You can install Dapr runtime in airgap (offline) environment using a pre-downloaded [installer bundle](https://github.com/dapr/installer-bundle/releases). You need to download the archived bundle for your OS beforehand (e.g., daprbundle_linux_amd64.tar.gz,) and unpack it. Thereafter use the `--from-dir` flag in the init command to point to the extracted bundle location to initialize Dapr.

```bash
# Initializing dapr in airgap environment

> dapr init --from-dir <path-to-bundle-location>

Author: Updated!

```

> Note: Assuming extracted bundle directory path is $HOME/daprbundle, <path-to-bundle-location> should be $HOME/daprbundle.

> Note: Dapr Installer bundle just contains the placement container apart from the binaries and so `zipkin` and `redis` are not enabled by default. You can pull the images locally either from network or private registry and run as follows:

> ```bash

Author: Done

```bash
# Example of pulling images from a private registry.
dapr init --image-registry example.io/<username>
```

#### Install in airgap environment

> You can install Dapr runtime in airgap (offline) environment using a pre-downloaded [installer bundle](https://github.com/dapr/installer-bundle/releases). You need to download the archived bundle for your OS beforehand (e.g., daprbundle_linux_amd64.tar.gz,) and unpack it. Thereafter use the local Dapr CLI binary in the bundle with `--from-dir` flag in the init command to point to the extracted bundle location to initialize Dapr.

Reviewer: ```suggestion
You can install Dapr runtime in airgap (offline) environment using a pre-downloaded [installer bundle](https://github.com/dapr/installer-bundle/releases). You need to download the archived bundle for your OS beforehand (for example daprbundle_linux_amd64.tar.gz for Linux,) and unpack it. Then use the local Dapr CLI binary included in the bundle with `--from-dir` flag with the `init` command to point to the extracted bundle location to initialize Dapr.
```

Move to the bundle directory and run the following command:

```bash

> # Initializing dapr in airgap environment

Reviewer: ```suggestion
# Initializing dapr in airgap Linux environment
```

./dapr init --from-dir .
```

> > For windows, use `.\dapr.exe` to point to the local Dapr CLI binary.

Reviewer: ```bash
# Initializing dapr in airgap Windows environment
.\dapr.exe init --from-dir .
```

Reviewer: Suggest making this a cut and paste for Windows, rather than just a comment. Makes the Windows users feel more inclusive!

> > For windows, use `.\dapr.exe` to point to the local Dapr CLI binary.

Reviewer: ```suggestion
```

> If you are not running the above command from the bundle directory, provide the full path to bundle directory as input. For example, assuming the bundle directory path is $HOME/daprbundle, run `$HOME/daprbundle/dapr init --from-dir $HOME/daprbundle` to have the same behavior.

> > Note: Dapr Installer bundle just contains the placement container apart from the binaries and so `zipkin` and `redis` are not enabled by default. You can pull the images locally either from network or private registry and run as follows:

Reviewer: ```suggestion
> Note: The Dapr Installer bundle just contains the Placement server container and the binaries. Zipkin and Redis containers are not included and therefore are not enabled by default. You can pull the images for these locally either from network or a private registry and run as follows:
```

```bash
docker run --name "dapr_zipkin" --restart always -d -p 9411:9411 openzipkin/zipkin
docker run --name "dapr_redis" --restart always -d -p 6379:6379 redis
```

> Alternatively to the above, you can also have slim installation as well to install dapr without running any Docker containers in airgap mode.   

Reviewer: ```suggestion
Alternatively you can also use the `slim` installation to install Dapr without running any Docker containers.   
```

$ dapr init --image-registry example.io/<username>
```

#### Install in airgap environment

> You can install Dapr runtime in airgap(offline) environment using pre-downloaded [installer bundle](https://github.com/dapr/installer-bundle/releases). You need to download the archived bundle beforehand (e.g. daprbundle_linux_amd64.tar.gz, daprbundle_windows_amd64.zip) and unpack it. Thereafter use the `--from-dir` flag in init command pointing to the bundle location to initialize dapr.

Reviewer 2: ```suggestion
You can install Dapr runtime in airgap (offline) environment using a pre-downloaded [installer bundle](https://github.com/dapr/installer-bundle/releases). You need to download the archived bundle for your OS beforehand (e.g., daprbundle_linux_amd64.tar.gz,) and unpack it. Thereafter use the `--from-dir` flag in the init command to point to the bundle location to initialize dapr.
```

Author: Done

```bash
# Initializing dapr in airgap environment

> $ dapr init --from-dir <path-to-bundle-location>

Reviewer 2: ```suggestion
dapr init --from-dir /path/to/bundle/location
```

Author: Done

```

> Note: Assuming extracted bundle directory path is $HOME/daprbundle, <path-to-bundle-location> should be $HOME/daprbundle.

> Note: Dapr Installer bundle just contains the placement container apart from the binaries and so `zipkin` and `redis` are not enabled by default. You can pull the images locally either from network or private registry and run as follows:
```bash

> 1. docker run --name "dapr_zipkin" --restart always -d -p 9411:9411 openzipkin/zipkin

Reviewer 2: Remove numbers as it's inside code block.

Author: Done, Thanks

2. docker run --name "dapr_redis" --restart always -d -p 6379:6379 redis
```

Alternatively to the above, you can also have slim installation as well to install dapr without running any Docker containers in airgap mode.
```bash

> $ dapr init --slim --from-dir <path-to-bundle-location>

Reviewer 2: Remove dollar

Author: Dollar sign is there just to have similarity with other flag descriptions:
$ dapr init --image-registry example.io/<username>
$ dapr init --runtime-version 1.0.0
$ docker network create dapr-network
etc.

Reviewer 2: Ah, the dollar sign breaks the copy functionality on the top right. We should ideally remove it from everywhere. 
![image](https://user-images.githubusercontent.com/22132944/160647197-07847301-954a-44a8-9525-46f79b3a482d.png)

Author: Done!
